src/main/java/no/nav/veilarbperson/utils/PersonV2DataMapper.java

```diff
@@ -112,4 +118,16 @@ public static Telefon telefonNummerMapper(HentPdlPerson.Telefonnummer telefonnum
                  : null;
     }
 
+    public static List<HentPdlPerson.VergemaalEllerFremtidsfullmakt> vergemaalMapper(List<HentPdlPerson.VergemaalEllerFremtidsfullmakt> vergemaalEllerFremtidsfullmaktListe) {
+        if (!vergemaalEllerFremtidsfullmaktListe.isEmpty()) {
+            for (HentPdlPerson.VergemaalEllerFremtidsfullmakt vergemaalEllerFremtidsfullmakt : vergemaalEllerFremtidsfullmaktListe) {
```

Comment:
Kan endre med funksjonell programmering. Hvis er det brukt flere steder så kan man refaktor denne kode til Util class.

kansje vi kan bruke denne lenke å få forskjell mellom for loop og streams. https://blog.jdriven.com/2019/10/loop/

Reply:
Jeg tror vi skal ha en egen common funksjonalitet å lage flere struktur fra PDL som kan konsumeres til de andre prosjekt. I denne common funksjonalitet man kan lage en bibliotek å gjenbruke flere struktur og kode med generics og funksjonell programmering osv..

Reply:
> Jeg tror det hadde vært greit å ha et eget endepunkt for verge/fullmakt istedenfor å slå det sammen med personalia.

Jeg tror det også. Flott:) Jeg tenker at kanskje vi må lage et ny endepunkt som ikke bare henter vergemål/fullmakt men også fornavn,mellomnavn,etternavn,fodselsdato,dodsdato,diskresjonskode,sikkerhetstiltak,egenansatt,kjonn til veilarbvisitkortfs. Tenker å lage en ny klass som skal inneholde all disse elementer i veilarbperson og det nye endepunktet skal returnere den nye klass. Tenker PersonV2data har alt for mye data og da trenger ikke å returnere den til frontend appen hvis frontend appen trenger ikke det. Hva synes du?